source/_components/sensor.huawei_lte.markdown

```diff
@@ -33,19 +33,40 @@ sensor:
       - traffic_statistics.TotalConnectTime
 ```
 
-Configuration variables:
-
-**monitored_conditions** array (*Optional*): Defines the data to monitor as sensors. Defaults to a few generally available data items expected to be available on most boxes. The names here are dot separated paths to information returned by the router. The data set varies by router model; to see what your router provides, set logging level to debug and watch homeassistant.components.huawei_lte debug entries. The following list contains a few example paths just to illustrate the syntax; these may not be available on all routers or their semantics may differ, and there are quite likely many more that are not listed here.
-
-  - **device_information.SoftwareVersion**: Software version.
-  - **device_information.WanIPAddress**: WAN interface IP address.
-  - **device_information.WanIPv6Address**: WAN interface IP address.
-  - **device_signal.rsrq**: The signal RSRQ value.
-  - **device_signal.rsrp**: The signal RSRP value.
-  - **device_signal.rssi**: The signal RSSI value.
-  - **device_signal.sinr**: The signal SINR value.
-  - **traffic_statistics.CurrentDownloadRate**: Current download rate, bytes/sec.
-  - **traffic_statistics.CurrentUploadRate**: Current upload rate, bytes/sec.
-  - **traffic_statistics.TotalUpload**: Total bytes uploaded since last reset.
-  - **traffic_statistics.TotalDownload**: Total bytes downloaded since last reset.
-  - **traffic_statistics.TotalConnectTime**: Total time connected since last reset.
+{% configuration %}
+monitored_conditions:
+  description: Defines the data to monitor as sensors. The names here are dot separated paths to information returned by the router. The data set varies by router model; to see what your router provides, set logging level to debug and watch `homeassistant.components.huawei_lte` debug entries. The following list of values contains a few example paths just to illustrate the syntax; these may not be available on all routers or their semantics may differ, and there are quite likely many more that are not listed here.
+  type: list
+  required: false
+  default:
```

Comment:
Not sure if the following list is the best solution, it renders bad on the documentation page in my opinion.

> Default value: [“device_information.WanIPAddress”, “device_signal.rsrq”, “device_signal.rsrp”, “device_signal.rssi”, “device_signal.sinr”]

As you can see it uses curved quotation marks.

What about adding it like
`default: ['device_information.WanIPAddress', 'device_signal.rsrq', 'device_signal.rsrp', 'device_signal.rssi', 'device_signal.sinr']`
to avoid this?

(might need to encapsulate the square brackets with with `"`, not sure)

Reply:
In my opinion, if list rendering is not working satisfactorily, it should be fixed at the source, i.e. in whatever does that rendering. Working around it like that doesn't sound too good to me.

Reply:
Totally agree, otherwise this will just happen again.

Reply:
@tjorim IMHO there is nothing wrong with the rendering there.

Although I would prefer single quotes personally as well, this still is perfectly valid. Feel free to open a discussion elsewhere.